main.tf

```diff
@@ -1,21 +1,16 @@
 module "label" {
```

Comment:
@syphernl I believe you don't need the label anymore. You can use module.this everywhere, since you don't have multiple labels.